### DISCUSSION TITLE
AG eating up my hard drive

### DISCUSSION THREAD

**Discussion by Umpalumpe** [apohl79/audiogridder#454] [Help]
Starting the day I only had 35GB left on my system drive (mac) and the only thing I've been doing today is to setup up a mixing template in Ableton using Audiogridder on heavy plugins on my master and groups/busses, but now my hard drive have ran out of space for some reason and I'm not able to track down what's been going on and where all my 35GB have gone. So I've been trying to wipe cache & restart in Audiogridder now, but it doesn't look like it's helping. And as said, I haven't done anything other today, than work in Ableton with Audiogridder just to setup up a template, so no audio involved.

**Comment by Umpalumpe** [comment]
So if anybody else have experienced the same thing and are able to help me, that would be much appreciated!?

**Comment by apohl79** [comment]
Check if this discussion helps: https://github.com/apohl79/audiogridder/discussions/390

**Reply by Umpalumpe** [reply]
That might be helpful. Cause I do have 80GB of data in the /cores folder. But is it safe to delete them all?

**Reply by Umpalumpe** [reply]
Okay, now I saw the load more button, to see the whole thread. github noob. Thanks!
